xml/glossary.xml

```diff
@@ -10,9 +10,9 @@
  the other terms it might make more sense to include the upstream glossary that is appended to all
  upstream manuals -->
 <glossary xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="gl.cloud">
- <title>Terminology</title>
- <info>
-<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+    <title>Terminology: A glossary of terms and product names</title>
```

Comment:
Good idea! But I think the subtitle needs title style  to match the style guide.